website/docs/guides/version-3-upgrade.html.md

```diff
@@ -615,6 +616,29 @@ resource "aws_autoscaling_group" "example"{
 }
 ```
 
+## Resource: aws_cloudfront_distribution
+
+### active_trusted_signers Attribute Name and Type Change
+
+Previously, the `active_trusted_signers` computed attribute was implemented with a Map that did not support accessing its computed `items` attribute in Terraform 0.12. 
+To address this, while also enabling access to this data without a state migration, the `active_trusted_signers` attribute has been renamed to `trusted_signers` and is now implemented as a List with a computed `items` List attribute and computed `enabled` boolean attribute. 
```

Comment:
Practitioners won't need to know about the "state migration" details, so probably best to omit that

```suggestion
To address this, the `active_trusted_signers` attribute has been renamed to `trusted_signers` and is now implemented as a List with a computed `items` List attribute and computed `enabled` boolean attribute. 
```